solutions/client-landing-zone/client-folder/standard/applications-infrastructure/nonp/firewall-policy/rules/defaults.yaml

```diff
@@ -0,0 +1,128 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+# Rules 2147483541 to 2147483546 are the suggested defaults by Google
+#########
+# Delegate to host project, egress traffic(firewall) from VPC resources to private IP ranges in shared VPC network
+# AC-3(9), AC-4, AC-4(21), SC-7(5), SC-7(8), SC-7(9), SC-7(11) - All connections to or from virtual machine instances are allowed/denied via firewall rules configured in shared VPC network within host project or firewall policies in parent folders based on least-privilege principle. Each firewall rule applies to incoming(ingress) or outgoing(egress) connections, not both.
+# AU-12 - Enable Logging for firewall policies
+# SI-4 - Logging denied traffic
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeFirewallPolicyRule
+metadata:
+  name: client-name-standard-app-infra-nonp-fwpol-exclude-private-ip-ranges-egress-fwr # kpt-set: ${client-name}-standard-app-infra-nonp-fwpol-exclude-private-ip-ranges-egress-fwr
```

Comment:
Have you tested these? Without the rendered client-name, client-name-standard-app-infra-nonp-fwpol-exclude-private-ip-ranges-egress-fwr is 78 characters long.

Reply:
yes I tested it and it deployed ok (I purposely shortened "applications-infrastructure" to "app-infra" in the name to keep it as short as possible)
for example, that same rule already exists on the "applications-infrastructure" folder and is 91 characters before rendering